### PR TITLE
feat: add notification settings screen

### PR DESCRIPTION
**Pull request by UncleSamtoshi**
- add screen to set notification settings
- unify icons in settings to all be outlines

### CODE REVIEW

File: app/screens/settings-screen/notifications-screen.tsx [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+  NotificationSettings,
+  useAccountDisableNotificationCategoryMutation,
+  useAccountDisableNotificationChannelMutation,
+  useAccountEnableNotificationCategoryMutation,
+  useAccountEnableNotificationChannelMutation,
+  useNotificationSettingsQuery,
+} from "@app/graphql/generated"
+import { useIsAuthed } from "@app/graphql/is-authed-context"
+
+const useStyles = makeStyles(() => ({
```
Comment (nicolasburtey) [inline app/screens/settings-screen/notifications-screen.tsx:25]: our pattern has been to put that style definition at the end of the file

**Review by nicolasburtey — COMMENTED** [review]
some cosmetics but LGTM otherwise

File: app/screens/settings-screen/notifications-screen.tsx (hunk 1 above)
Comment (UncleSamtoshi) [reply app/screens/settings-screen/notifications-screen.tsx:25]: Moved it

File: app/screens/settings-screen/notifications-screen.tsx [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+    },
+  )
+
+  return (
+    <Screen style={styles.container} preset="scroll">
+      <View style={styles.settingsHeader}>
+        <Switch
+          value={pushNotificationsEnabled}
+          onValueChange={async (value) => {
+            if (value) {
```
[the remark this answers is not in the document]
Comment (UncleSamtoshi) [reply app/screens/settings-screen/notifications-screen.tsx:263]: Will make this change

**Review by nicolasburtey — APPROVED** [review]

**Pull request merged**